src/unix/linux_like/linux/mod.rs

```diff
@@ -5620,6 +5620,8 @@ extern "C" {
         len: ::size_t,
         flags: ::c_uint,
     ) -> ::ssize_t;
+
+    pub fn klogctl(syslog_type: ::c_int, buf: *mut ::c_char, len: ::c_int) -> ::c_int;
```

Comment:
Nit: Linux docs call the second arg `bufp` per https://linux.die.net/man/3/klogctl